### DISCUSSION TITLE
Where would you like to travel?

### DISCUSSION THREAD

**Discussion by marshr00m** [NAIST-SE/AttractingContributors#1190] [2024 Attraction Contributions]
Let me know your thoughts!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel

**Comment by masataka-ki** [comment]
I've sent you a pull request, so if you want, please pull request for me too!
https://github.com/masataka-ki/AttractingContributors-Please-tell-me-a-country-you-want-to-visit-in-the-future
